packages/client/src/ActionsAndReducers/playerUi/helpers/handleForceDelta.js

```diff
@@ -0,0 +1,39 @@
+
+import handleVisibilityChanges from './handleVisibilityChanges'
+import handlePerceptionChange from './handlePerceptionChanges'
+import handleStateOfWorldChanges from './handleStateOfWorldChanges'
+import handleForceLaydownChanges from './handleForceLaydownChanges'
+import handlePlansSubmittedChanges from './handlePlansSubmittedChanges'
+import {
+  FORCE_LAYDOWN,
+  VISIBILIY_CHANGES,
+  PERCEPTION_OF_CONTACT,
+  SUBMIT_PLANS,
+  STATE_OF_WORLD
+} from '../../../consts'
+
+/** create a marker for the supplied set of details */
+export default (/* object */message, /* object */ allForces) => {
+  console.log('message', message)
+  const msgType = message.details.messageType
+  if (!msgType) {
```

Comment:
Is it possible we can move away from console.logging errors and actually use `throw`?

Reply:
Oh dear - I should have removed it.  Googling `throw`

Reply:
Ah, I wasn't aware of `throw`. I see we'll need another level of infrastructure, to contain the `catch` elements.

Reply:
You don't HAVE to use a `catch` element to use `throw`, it's just that without it the program will terminate. Sometimes that is the desired result, sometimes it's not.